osu.Game/Overlays/Settings/SettingsItem.cs

```diff
@@ -57,6 +59,19 @@ public virtual LocalisableString LabelText
             }
         }
 
+        /// <summary>
+        /// Text to be displayed at the bottom of this <see cref="SettingsItem{T}"/>.
+        /// Used for further explanation or indicating drawbacks of the current setting.
+        /// </summary>
+        public string NoteText
+        {
+            set
+            {
+                noteText.Alpha = 1;
```

Comment:
Would maybe make this

```suggestion
                noteText.Alpha = string.IsNullOrWhiteSpace(value) ? 0 : 1;
```

because it seems weird that these start hidden by default but then setting empty string sets alpha to 1.

Also not super sure about no fade in/out but maybe easier to skip that for now.